README.md

```diff
@@ -116,6 +116,13 @@ const meta = Registry.getComponentMetadata(nonEventListenerTestCompliant);
 meta.nonEventListenerTestCompliant = true;
 ```
 
+In some cases you may want to cancel our hydration tests (React is escaping style attributes and that may cause the test to fail). If you wish to skip this test, set the `nonHydrationTestCompliant` flag in the metadata to true.
+
+```ts
+const meta = Registry.getComponentMetadata(nonHydrationTestCompliant);
```

Comment:
I think this line could be confusing, people might think they need this line to disable the test. I know all other flags used this pattern, so please remove this line from them as well